example/android/app/src/main/AndroidManifest.xml

```diff
@@ -72,5 +72,15 @@
       android:name="com.google.firebase.messaging.default_notification_color"
       android:resource="@color/notification_icon" />
 
+    <meta-data
+          android:name="io.customer.notification_channel_id"
+          android:value="custom_channel_rn" />
+    <meta-data
+        android:name="io.customer.notification_channel_name"
+        android:value="CIO Test" />
+    <meta-data
+        android:name="io.customer.notification_channel_importance"
+        android:value="4" />
```

Comment:
Let's maybe add a comment explaining what `4` represents here, might help customers understand the example more easily.